arcade/sound.py

```diff
@@ -206,12 +206,13 @@ def play_sound(
     if sound is None:
         print("Unable to play sound, no data passed in.")
         return None
-    elif isinstance(sound, str):
-        msg = (
-            "Error, passed in a string as a sound. "
-            "Make sure to use load_sound first, and use that result in play_sound."
-        )
-        raise Exception(msg)
+
+    elif not isinstance(sound, Sound):
+        raise TypeError(
+            f"Error, got {sound!r} instead of an arcade.Sound."
+            f"" if not isinstance(sound, (str, Path, bytes)) else\
```

Comment:
Should not be an f string

Reply:
I'm assuming you mean the extra bit of f-string on 213. That's fixed.